editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -844,6 +844,11 @@ export const NavigatorItem: React.FunctionComponent<
     ? 'component'
     : resultingStyle.iconColor
 
+  const currentlyRenaming = useMemo(
+    () => EP.pathsEqual(props.renamingTarget, props.navigatorEntry.elementPath),
+    [props.renamingTarget, props.navigatorEntry.elementPath],
+  )
+
```

Comment:
I'm not sure it is worth using useMemo for this simple equality check, but if have this, let's use it in line 1060 too, to pass it in as a prop to ItemLabel.

Reply:
nope it's not important to use a useMemo for this! 
1. the result is a boolean so it will be value compared so we don't need to worry about losing reference equality
2. ElementPath is very heavily memoized and pathsEqual is probably faster than react's memo check

Reply:
good point, removed the unnecessary `useMemo`